docs/userguide/networking/default_network/container-basics.md

```diff
@@ -40,7 +40,7 @@ The steps with which Docker configures a container are:
 
 With these steps complete, the container now possesses an `eth0` (virtual) network card and will find itself able to communicate with other containers and the rest of the Internet.
 
-You can opt out of the above process for a particular container by giving the `--net=` option to `docker run`, which takes four possible values.
+You can opt out of the above process for a particular container by giving the `--net=` option to `docker run`, which takes five possible values.
```

Comment:
`which takes these values:`   is cleaner and requires less maintenance than counting